addons/hr_holidays/views/hr_leave_type_views.xml

```diff
@@ -78,7 +78,7 @@
                             <field name="allocation_validation_type" string="Approval" widget="radio" attrs="{'invisible': ['|', ('requires_allocation', '=', 'no'), ('employee_requests', '=', 'no')]}"/>
                         </group>
                     </group>
-                    <group name="visual" id="visual" >
+                    <group name="visual" id="visual" class="flex-column">
```

Comment:
This is a very strange template 🤔
I'm not sure why it has been coded with an empty group, it may be a mistake or a leftover from previous implementations...

Rather then adding the class, could you remove the first  `<group colspan="2"` and apply the `string="Display Option"` attribute to the second group  instead?  thanks